src/ser.rs

```diff
@@ -32,3 +57,29 @@ impl Serialize for chrono::Duration {
         serializer.serialize_i64(nanoseconds)
     }
 }
+
+#[cfg(feature = "chrono")]
+impl Serialize for Option<chrono::Duration> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        struct Data<'a, V: 'a>(&'a V)
+        where
+            V: Serialize;
```

Comment:
I forget you can do this in a function. Nice addition. Definitely makes this read much cleaner 😄